models/release_image.go

```diff
@@ -24,6 +24,9 @@ type ReleaseImage struct {
 	// Required: true
 	CPUArchitecture *string `json:"cpu_architecture" gorm:"default:'x86_64'"`
 
+	// List of CPU architectures provided by the image.
+	CPUArchitectures []string `json:"cpu_architectures"`
```

Comment:
Should we update default_release_images.json file with cpu_architectures as well? Is it the next step to have cpu_architectures required and remove/deprecate cpu_architecture?

Reply:
That's a very good suggestion, just added it.

By the way, given that the code detects it automatically, I wonder whether we even need CPU arch(s) and release version in this file at all... Any thoughts?

Reply:
Maybe @osherdp can suggest?

The current entry looks like
```
    {
        "openshift_version": "4.11",
        "cpu_architecture": "arm64",
        "cpu_architectures": ["arm64"],
        "url": "quay.io/openshift-release-dev/ocp-release:4.11.0-rc.7-aarch64",
        "version": "4.11.0-rc.7",
        "support_level": "beta"
    }
```

but given the implementation that handles ClusterImageSet containing only the URL (so we have capability to detect CPU arch and version), it looks to me like some of those here are redundant

Reply:
Forget, after analyzing stuff we cannot remove anything and we must keep both `arch` and `archs`. Sorry for the noise